Source/Core/InputCommon/ControllerEmu/StickGate.cpp

```diff
@@ -138,6 +175,11 @@ ControlState ReshapableInput::GetInputRadiusAtAngle(double angle) const
   return GetCalibrationDataRadiusAtAngle(m_calibration, angle);
 }
 
+ControlState ReshapableInput::GetVirtualNotchDegrees() const
```

Comment:
Please make this function return radians. Any use of this value in code will need to be in radians.
Only the UI need be in degrees for mortals/users.